src/main/java/astvisitors/JasminCodeGeneratorVisitor.java

```diff
@@ -0,0 +1,684 @@
+		append(node.getId());
+		append("(");
+		for (AExpr parameter : node.getParameters()) {
+            switch(parameter.getType()) {
+            case INT: case BOOL:
+            	append("I");
+            	break;
+            case DOUBLE:
+            	append("D");
+            	break;
+            case STRING:
+            	append("Ljava/lang/String;");
+            	break;
+            }
+            ;
+        }
+        append(")");
+        switch(node.getType()) {
+        case INT: case BOOL:
+        	appendLine("I");
+        	break;
+        case DOUBLE:
+        	appendLine("D");
+        	break;
+        case STRING:
+        	appendLine("Ljava/lang/String;");
+        	break;
+        case VOID:
+        	appendLine("V");
+        	break;
+        }
+ 
+	}
+	private void generateSignatureForFuncDef(Func_defNode node) {
+		append(node.getId());
+		append("(");
+		for (DclNode parameter : node.getParameters()) {
+            switch(parameter.getType()) {
+            case INT: case BOOL:
+            	append("I");
+            	break;
+            case DOUBLE:
+            	append("D");
+            	break;
+            case STRING:
+            	append("Ljava/lang/String;");
+            	break;
+            }
+            ;
+        }
+        append(")");
+        switch(node.getType()) {
+        case INT: case BOOL:
+        	appendLine("I");
+        	break;
+        case DOUBLE:
+        	appendLine("D");
+        	break;
+        case STRING:
+        	appendLine("Ljava/lang/String;");
+        	break;
+        case VOID:
+        	appendLine("V");
+        	break;
+        }
+		
+	}
+	private void generateSignatureForExprFunc(Func_callExprNode node) {
+		append(node.getID());
+		append("(");
+		for (AExpr parameter : node.getParameters()) {
+            switch(parameter.getType()) {
+            case INT: case BOOL:
+            	append("I");
+            	break;
+            case DOUBLE:
+            	append("D");
+            	break;
+            case STRING:
+            	append("Ljava/lang/String;");
+            	break;
+            }
+            ;
+        }
+        append(")");
+        switch(node.getType()) {
+        case INT: case BOOL:
+        	appendLine("I");
+        	break;
+        case DOUBLE:
+        	appendLine("D");
+        	break;
+        case STRING:
+        	appendLine("Ljava/lang/String;");
+        	break;
+        case VOID:
+        	appendLine("V");
+        	break;
+        }
+	}
+	private void generatePrefixSetupCode() {
+		appendLine(".class public " + programName);
+		appendLine(".super java/lang/Object");
+		appendLine(".method public static main([Ljava/lang/String;)V");
+	}
+	private void generatePostfixSetupCode() {
+        appendLine("return");
+        appendLine(".limit stack " + maxStackSize);
+        appendLine(".limit locals " + maxLocals);
+        appendLine(".end method");
+        sb.append(functionStringBuilder);
+	}
```

Comment:
Maybe add function which takes a `Type` and returns JVM Type? 
When doing that, you can just concat strings together to create the JVM instruction